specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/vaults.json

```diff
@@ -419,89 +419,78 @@
       "type": "object",
       "properties": {
         "Provider": {
-          "description": "Name of the resource provider.",
-          "type": "string",
-          "readOnly": true
+          "description": "Name of the provider for display purposes",
+          "type": "string"
         },
         "Resource": {
-          "description": "Type of the resource.",
-          "type": "string",
-          "readOnly": true
+          "description": "Name of the resource type for display purposes",
+          "type": "string"
         },
         "Operation": {
-          "description": "Name of the operation.",
-          "type": "string",
-          "readOnly": true
+          "description": "Name of the operation for display purposes",
+          "type": "string"
         },
         "Description": {
-          "description": "Operation description.",
-          "type": "string",
-          "readOnly": true
+          "description": "Description of the operation for display purposes",
+          "type": "string"
         }
       }
     },
     "ClientDiscoveryForLogSpecification": {
-      "description": "Log specification for the operation.",
+      "description": "Class to represent shoebox log specification in json client discovery.",
       "type": "object",
       "properties": {
         "name": {
-          "description": "Name of the log.",
-          "type": "string",
-          "readOnly": true
+          "description": "Name",
+          "type": "string"
         },
         "displayName": {
-          "description": "Localized display name.",
-          "type": "string",
-          "readOnly": true
+          "description": "Localized display name",
+          "type": "string"
         },
         "blobDuration": {
-          "description": "Blob duration.",
-          "type": "string",
-          "readOnly": true
+          "description": "blob duration",

```

Comment:
Hi @veronicagg , As this field is in Response and Shoebox creates blobs in customer storage account every hour. The value PT1H indicates blobs per hour. Thanks!